src/nyc_trees/apps/home/urls.py

```diff
@@ -17,7 +17,9 @@
     url(r'^jobs/(?P<job_id>\d+)/$', r.retrieve_job_status,
         name='retrieve_job_status'),
 
-    url(r'^training/$', r.training_list_page),
+    url(r'^training/$', r.training_list_page, name="training_list_page"),
+
+    url(r'^training/groups_to_follow/$', r.groups_to_follow),
```

Comment:
I could reproduce, and I get it now. I spoke to soon. this is kind of a flat pages issue, because flat pages is not requiring trailing slashes in their pattern, you never bottom out to the `APPEND_SLASH` redirect.
https://docs.djangoproject.com/en/1.7/ref/contrib/flatpages/#using-the-urlconf